<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1045</width>
                <height>735</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QPushButton" name="pushButtonAdd">
                <property name="geometry">
                    <rect>
                        <x>900</x>
                        <y>470</y>
                        <width>121</width>
                        <height>61</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Добавить Запись</string>
                </property>
            </widget>
            <widget class="QPushButton" name="allDataViewButton">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>570</y>
                        <width>121</width>
                        <height>61</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Вывести все</string>
                </property>
            </widget>
            <widget class="QPushButton" name="filterDataButton">
                <property name="geometry">
                    <rect>
                        <x>160</x>
                        <y>570</y>
                        <width>121</width>
                        <height>61</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Курс &gt; 2</string>
                </property>
            </widget>
            <widget class="QPushButton" name="filterData2Button">
                <property name="geometry">
                    <rect>
                        <x>310</x>
                        <y>570</y>
                        <width>121</width>
                        <height>61</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Курс &lt;= 2</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButtonRem">
                <property name="geometry">
                    <rect>
                        <x>900</x>
                        <y>620</y>
                        <width>121</width>
                        <height>61</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Удалить запись</string>
                </property>
            </widget>
            <widget class="QLabel" name="surname_label">
                <property name="geometry">
                    <rect>
                        <x>790</x>
                        <y>20</y>
                        <width>111</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>Surname</string>
                </property>
            </widget>
            <widget class="QTableView" name="tableView">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>20</y>
                        <width>691</width>
                        <height>511</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="university_label">
                <property name="geometry">
                    <rect>
                        <x>790</x>
                        <y>130</y>
                        <width>121</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>University</string>
                </property>
            </widget>
            <widget class="QLabel" name="id_label">
                <property name="geometry">
                    <rect>
                        <x>780</x>
                        <y>500</y>
                        <width>91</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>ID</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEditID">
                <property name="geometry">
                    <rect>
                        <x>790</x>
                        <y>560</y>
                        <width>231</width>
                        <height>41</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEditUniversity">
                <property name="geometry">
                    <rect>
                        <x>790</x>
                        <y>190</y>
                        <width>231</width>
                        <height>41</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEditName">
                <property name="geometry">
                    <rect>
                        <x>790</x>
                        <y>70</y>
                        <width>231</width>
                        <height>41</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEditYear">
                <property name="geometry">
                    <rect>
                        <x>790</x>
                        <y>410</y>
                        <width>231</width>
                        <height>41</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="year_label">
                <property name="geometry">
                    <rect>
                        <x>790</x>
                        <y>350</y>
                        <width>91</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>Year</string>
                </property>
            </widget>
            <widget class="QLabel" name="faculty_label">
                <property name="geometry">
                    <rect>
                        <x>790</x>
                        <y>240</y>
                        <width>91</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>Faculty</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEditFaculty">
                <property name="geometry">
                    <rect>
                        <x>790</x>
                        <y>290</y>
                        <width>231</width>
                        <height>41</height>
                    </rect>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButtonSort">
                <property name="geometry">
                    <rect>
                        <x>470</x>
                        <y>572</y>
                        <width>181</width>
                        <height>61</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Сортировать по курсу</string>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1045</width>
                    <height>20</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuLab_7">
                <property name="title">
                    <string>Lab 7</string>
                </property>
            </widget>
            <addaction name="menuLab_7"/>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
